documentation/modules/upgrading/proc-downgrade-kafka-kraft.adoc

```diff
@@ -0,0 +1,80 @@
+// This module is included in the following assemblies:
+//
+// assembly-downgrade.adoc
+
+[id='proc-downgrade-kafka-kraft-{context}']
+= Downgrading KRaft-based Kafka clusters and client applications
+
+[role="_abstract"]
+Downgrade a KRaft-based Strimzi Kafka cluster to an earlier version.
+When downgrading a KRaft-based Strimzi Kafka cluster to a lower version, like moving from {KafkaVersionHigher} to {KafkaVersionLower}, ensure that the `spec.kafka.metadataVersion` property in the Kafka configuration aligns with a version supported by the specified Kafka version (`spec.kafka.version`). 
```

Comment:
I would write this more generic -> this is not about what is set in the custom resource but what the cluster actually uses. For example, if you change the `.spec.kafka.metadataVerison` to `3.5`, but the Kafka cluster does not accept it, the downgrade still cannot be done.

```suggestion
When downgrading a KRaft-based Strimzi Kafka cluster to a lower version, like moving from {KafkaVersionHigher} to {KafkaVersionLower}, ensure that the metadata version used by the Kafka cluster is a version supported by the Kafka version you want to downgrade to. 
```